testsuite/gna/bug097/testsuite.sh

```diff
@@ -2,17 +2,19 @@
 
 . ../../testenv.sh
 
-if [ -z $CC ]; then
-  CC="gcc"
-fi
+if c_compiler_is_available; then
```

Comment:
Please, echo a message and `exit 0`, instead of wrapping all the commands. See https://github.com/ghdl/ghdl/blob/master/testsuite/gna/bug032/testsuite.sh#L3-L7